apps/google-analytics-4/frontend/src/components/sidebar/ChartHeader.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from 'react';
+import { Box, DisplayText, Flex, Paragraph, Select } from '@contentful/f36-components';
+
+interface Props {
+  metricName: string;
+  metricValue: string;
+}
+
+export default function ChartHeader(props: Props) {
+  const { metricName, metricValue } = props;
+
+  const [dateSelection, setDateSelection] = useState('lastWeek');
```

Comment:
nit: missing typing on useState ie - `useState<string>('lastWeek');`

Reply:
Or better yet - is there a Interface/Type for this? You definitely know this better than I do but are the date range options limited to like 
```
'last-week'
'current-week'
'one-month'
```
If so then we can just create a type/interface of DateRangeType so that you can only change to those values